tools-local/tasks/installer.tasks/BuildTools.Publish/CloudTestTasks/AzureHelper.cs

```diff
@@ -50,7 +50,7 @@ public enum SasAccessType
             string stringToSign = string.Format(
                 "{0}\n\n\n{1}\n{5}\n{6}\n\n\n{2}\n\n\n\n{3}{4}",
                 method,
-                (size == string.Empty) ? string.Empty : size,
+                (size.Length == 0) ? string.Empty : size,
```

Comment:
This looks strange (even the original code). Why substitute one empty string for another?

Reply:
@drewnoakes, Yes, I agree it's strange. I think just using `size` without any checks should be enough. Do you think I change it? or wait someone's to confirm?

Reply:
I would just revert it.

Reply:
@stephentoub, Why not just use `size`?
The ternary operator makes no sense here as pointed by @drewnoakes. Whether the condition is true or false, the result of the ternary operator would be `size`.

Reply:
I don't know what the original developer was thinking when they wrote this.  I agree it's probably a fine change to make, but there's no evidence that tests have run to validate the change, there's no evidence that the change has any impact on performance (it won't), and we don't like to take churn unless it's meaningful.

Reply:
This still needs to be changed.